src/js/shield_canvas_draw.js

```diff
@@ -693,14 +697,21 @@ export function draw(name, ctx, options, ref) {
 //Register draw functions
 const drawFunctions = {};
 
+//Register fixed width values
+const fixedWidthDefinitions = {};
```

Comment:
Ideally, these parallel structures would be folded into a single constant structured along these lines:

```js
{
  branson: {
    draw: function () { … },
    width: 20,
  },
};
```

The current code is fine, but consider that refactor down the line if we end up attaching more metadata to these custom drawing functions.